### PR TITLE
[WIP] Resolve imports and exports

### PR DESCRIPTION
**Pull request by kmdrGroch**
fixes: #2373

### CODE REVIEW

**Comment by kmdrGroch** [comment]
@nicojs It's mostly done, but to progress I need your help with resolving these problems since I have no idea what's an issue :c

File: packages/core/bin/stryker [hunk 1]
```diff
@@ -5,5 +5,5 @@
 process.title = 'stryker';
 
 // Run the Stryker CLI
-const StrykerCli = require('../src/StrykerCli').default;
+const { StrykerCli } = require('../src/StrykerCli');
```
Comment (nicojs) [inline packages/core/bin/stryker:8]: This was the issue. We used to export `StrykerCli` as default. Stryker compiled as `default` export. Now a named export.

**Comment by nicojs** [comment]
BTW, don't look into the green windows tests. This is something we need to fix still, see https://github.com/nicojs/node-link-parent-bin/issues/20

**Comment by kmdrGroch** [comment]
Since there are a lot of changes nowadays, I think I'll postpone this PR for later on (after a release most likely)

**Pull request closed without merging**